src/BleServer.cpp

```diff
@@ -134,14 +134,17 @@ void BleServer::onDisconnect(NimBLEServer *pServer) {
     NimBLEDevice::startAdvertising();
 }
 
+#ifdef CANBUS_ENABLED
 void BleServer::init(Stream *vesc, CanBus *canbus) {
+    this->canbus = canbus;
+#else
+void BleServer::init(Stream *vesc) {
+#endif
     vescSerial = vesc;
```

Comment:
**Suggestion:** The `init` function in `BleServer` class has been overloaded to support two different types of initialization. This could lead to confusion and potential bugs in the future. Consider refactoring this to use a single function with optional parameters.
```suggestion
void BleServer::init(Stream *vesc, CanBus *canbus = nullptr) {
    this->canbus = canbus;
    vescSerial = vesc;
}
```